response_operations_ui/controllers/admin_controller.py

```diff
@@ -15,13 +16,16 @@ def _get_redis():
     return r
 
 
-def set_banner(banner):
+def set_banner_and_time(banner, time):
```

Comment:
I think some of Warrens comments have been missed from the previous PR

Reply:
This change was made, however when we passed a default datetime.now parameter in it would only record the time it was first called and would not update when a banner was removed/updated. I don't know if this is something to do with redis?